tools/site_and_regional/run_neon.py

```diff
@@ -611,6 +615,9 @@ def run_case(
             case.check_all_input_data()
             if not setup_only:
                 case.submit(no_batch=no_batch)
+                logger.info("-----------------------------------")
+                logger.info("Successfully submitted case!")
+                logger.info("Use 'qstat -u <user-name>' to check its run status")
```

Comment:
The problem with this is that you are assuming that PBS is being used, which won't necessarily be the case on non NCAR machines. I think run-neon.py is agnostic in terms of machine, so we shouldn't make this assumption here. The build does actually specify what the status command is, and there is a script to check the status (although it was non functional the last time I looked). So I think with a little more work we could get the right name.

Or you could add something to the effect of

Use the batch status checker on your system to check it's run status

That's not nearly as good for a naive user though so...

Reply:
Yes true this does rely on having PBS.

Reply:
It looks like you just need to do this...

`./xmlquery batch_query`

to get the batch q-status. So that's easy enough.

There is also a ./case.qstatus script, but it doesn't seem to ever work correctly for me so I don't use it. We really should get that working in cime if it populates cases with it.

Reply:
@adrifoster and I met and worked out what this needed to be and she implemented it. So this is resolved.